src/charm.py

```diff
@@ -223,14 +235,21 @@ def _integrations_config(self) -> dict:
         juju_application = self.model.app.name
         juju_unit = self.unit.name
 
+        job_name = f"juju_{juju_model}_{juju_model_uuid}_{juju_application}_self-monitoring"
```

Comment:
Safer to avoid f-strings just in case this ends up as a machine charm somewhere. It's not _really_ k8s specific, and could wind up on python 3.5 or something.

Reply:
This is not in a library so it will never end up on python 3.5.

Reply:
Ack. Python 3.5 was only mandatory for libraries. I meant for charms it is ok to use upto 3.8 if I am not mistaken.